frontend/src/concepts/pipelines/content/createRun/contentSections/TriggerTypeField.tsx

```diff
@@ -52,13 +54,14 @@ const TriggerTypeField: React.FC<TriggerTypeFieldProps> = ({ data, onChange }) =
             <SplitItem>
               <NumberInputWrapper
                 min={1}
-                value={parseInt(data.value) || 1}
-                onChange={(value) =>
+                value={numberPart}
+                onChange={(newNumberPart) => {
+                  const updatedValue = `${newNumberPart}${unitPart}`;
                   onChange({
                     ...data,
-                    value: replaceNumericPartWithString(data.value, value ?? 0),
```

Comment:
so a few things wrong here, once it is converted to a string, it cannot be edited unless you delete it. also when you submit when the units turn into something bigger it throws an error

Failed to create a recurring run due to conversion error: Invalid input error: Found invalid period schedule interval 0. Set at interval to least 1 second